### ISSUE TITLE
Reminders aren't persisting after bot restarts.

### ISSUE THREAD

**Issue by scragly** [python-discord/bot#363]
I haven't had a chance to look into it too deeply, but reminders are currently being lost as the expiry value of reminders aren't storing in the database. I assume this means that the reminder is taken to be expired already and so is purged.

**Comment by ilysia** [comment]
Hey! If this is still a problem I'd work on this

**Comment by lemonsaurus** [comment]
We're not really sure if this is still a problem. There have been major changes to the bug since it was last observed. And even if it is still a problem;
A) It's a bit of a [heisenbug](https://en.wikipedia.org/wiki/Heisenbug), so it'll be very hard to test if it's been solved.
B) The correct solution is probably a rewrite of the Reminder cog, so we can provide features like recurring reminders. 
C) There's been talk of putting some NLP into the rewrite so we can have stuff like `!remind "@Admins meeting in 5 minutes" every Sunday at 20:55`

Basically I don't think anyone should be working on this before we have determined if it's still happening and what the new design should be.

**Comment by scragly** [comment]
It appears this bug has been resolved after moving to the django branch. Closing due to this.
